### PR TITLE
[rum] Web view instrumentation for Mobile Session Replay

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds instructions on how to instrument web views for Mobile RUM SDK (iOS and Android only)
- Removes "web views" from list of unsupported components in Troubleshooting

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 1]
```diff
@@ -88,12 +91,52 @@ To set up Mobile Session Replay for iOS:
...
+## Web view instrumentation
+
+You can record the entire user journey across both [web and native views][1] on iOS or Android and watch it in a single Session Replay.
+
+The Session Replay is recorded through the Browser SDK, then the Mobile SDK handles the batching and uploading of the webview recording.
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+To instrument your consolidated web and native views Session Replay for Android:
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:103]:
```suggestion
To instrument your consolidated web and native Session Replay views for Android:
```

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 2]
```diff
@@ -88,12 +91,52 @@ To set up Mobile Session Replay for iOS:
...
+[1]: /real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking/
+[2]: https://github.com/DataDog/dd-sdk-ios/releases/tag/2.13.0
+[3]: /real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking/?tab=android#instrument-your-web-views
+[4]: /real_user_monitoring/session_replay/browser/#setup
+[5]: /real_user_monitoring/mobile_and_tv_monitoring/setup/android/
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+To instrument your consolidated web and native views Session Replay for iOS:
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:119]:
```suggestion
To instrument your consolidated web and native Session Replay views for iOS:
```

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 3]
```diff
@@ -88,12 +91,52 @@ To set up Mobile Session Replay for iOS:
...
+[4]: /real_user_monitoring/mobile_and_tv_monitoring/setup/ios
+
+{{% /tab %}}
+{{< /tabs >}}
+
 ## Additional configuration
 ### Set the sample rate for recorded sessions to appear
 
-Sample rate is a required parameter in Session Replay configuration. It must be a number between 0.0 and 100.0, where 0 means no replays will be recorded and 100 means all RUM sessions will contain replay.
+Sample rate is a required parameter in Session Replay configuration. It must be a number between 0.0 and 100.0, where 0 means no replays are be recorded and 100 means all RUM sessions contain replay.
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:137]:
```suggestion
Sample rate is a required parameter in Session Replay configuration. It must be a number between 0.0 and 100.0, where 0 means no replays are recorded and 100 means all RUM sessions contain replay.
```

**Review by maycmlee — APPROVED** [review]
Couple small suggestions, but approving!

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 4]
```diff
@@ -88,12 +91,52 @@ To set up Mobile Session Replay for iOS:
...
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+To instrument your consolidated web and native views Session Replay for Android:
+
+1. Ensure you are using version [`2.8.0`][2] or higher of the Android SDK.
+2. Enable [webview tracking][3] for your mobile application.
+3. Enable [Session Replay][4] for your web application.
+4. Enable [Session Replay][5] for your mobile application.
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:108]: There is no need for a link here, when reading this the user is actually on the right page

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 5]
```diff
@@ -88,12 +91,52 @@ To set up Mobile Session Replay for iOS:
...
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+To instrument your consolidated web and native views Session Replay for iOS:
+
+1. Ensure you are using version [`2.13.0`][1] or higher of the iOS SDK.
+2. Enable [webview tracking][2] for your mobile application.
+3. Enable [Session Replay][3] for your web application.
+4. Enable [Session Replay][4] for your mobile application.
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:124]: There is no need for a link here, when reading this the user is actually on the right page

**Review by MaelNamNam — CHANGES_REQUESTED** [review]
Two small comments about some links, but overall LGTM, thanks!

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md (hunk 4 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md]: updated, thanks!

**Review by MaelNamNam — APPROVED** [review]
LGTM, thanks!

**Pull request merged**